spec/features/category_spec.rb

```diff
@@ -1,10 +1,15 @@
 RSpec.feature "category" do
   fixtures :categories
-  let!(:category) { categories(:category) }
+  fixtures :carriers
   fixtures :users
+  
+  let!(:category) { categories(:category) }
+  let!(:carrier) { carriers(:carrier) } 
   let(:user) { users(:user) }
 
   before :each do
+    carrier.update_attribute :category_id, category.id
```

Comment:
Wdyt think about associating carrier to category inside the fixure...that way you don't have to `update` inside the `before` block.

```yaml
carrier:
  name: "test carrier"
  category: category     #<------
  item_id: "TEST1234"
  manufacturer: "babywearing"
  model: "test model"
  color: "blue"
  size: 5
```